src/hotspot/share/code/aotCodeCache.cpp

```diff
@@ -507,12 +508,12 @@ AOTCodeCache::AOTCodeCache(bool is_dumping, bool is_using) :
       return;
     }
     log_info (aot, codecache, init)("Loaded %u AOT code entries from AOT Code Cache", _load_header->entries_count());
-    log_debug(aot, codecache, init)("  Adapters: total=%u", _load_header->adapters_count());
-    log_debug(aot, codecache, init)("  Shared Blobs: total=%u", _load_header->shared_blobs_count());
-    log_debug(aot, codecache, init)("  C1 Blobs: total=%u", _load_header->C1_blobs_count());
-    log_debug(aot, codecache, init)("  C2 Blobs: total=%u", _load_header->C2_blobs_count());
-    log_debug(aot, codecache, init)("  Stubs:    total=%u", _load_header->stubs_count());
-    log_debug(aot, codecache, init)("  Nmethods: total=%u", _load_header->nmethods_count());
+    log_debug(aot, codecache, init)("  %s: total=%u", aot_code_entry_kind_name[AOTCodeEntry::Adapter], _load_header->adapters_count());
```

Comment:
You need to fix the exit trace at line 1300 onwards to call `aot_code_entry_kind_name()`